src/irc_aliases.lua

```diff
@@ -402,4 +405,59 @@ aliases['config%s+%S+%s+%S+%s*%S*'] = function (ms, c, t, msg, authed)
     end
 end
 
+aliases['whitelist%s+%S+%s*%S*'] = function (ms, c, t, msg, _, sender)
+    local _, _, action, nick = msg:find('whitelist%s+(%S+)%s*(%S*)')
+
+    if action == 'status' and nick == '' then
+        local result = ms.irc_quotegrabs.whitelist_status(sender) and 'opted-in' or 'opted-out'
+        ms.irc.privmsg(c, t, ('%s: you are %s for quotegrabs.'):format(sender, result))
+    elseif action == 'status' and nick ~= '' then
```

Comment:
the `and nick ~= ''` is unnecessary as the previous case rules out other possibilities.